eland/ml/pytorch/transformers.py

```diff
@@ -390,7 +387,7 @@ def _create_config(self):
                     "max_sequence_length"
                 ] = max_sequence_length
 
-        if self._traceable_model.classification_labels():
+        if isinstance(self._traceable_model, _TraceableClassificationModel):
```

Comment:
Are we losing a comparison here? Previously we also called `.classification_labels()`, do we need both or is the `isinstance` enough?

Reply:
I was running in to mypy errors but I've fixed that so I've reverted to the previous approach.